barretenberg/cpp/src/barretenberg/vm/avm/trace/public_inputs.hpp

```diff
@@ -7,12 +7,12 @@
 using FF = bb::AvmFlavorSettings::FF;
 
 struct EthAddress {
-    std::array<uint8_t, 20> value;
+    std::array<uint8_t, 20> value{};
 };
 
 struct Gas {
-    uint32_t l2_gas;
-    uint32_t da_gas;
+    uint32_t l2_gas{};
+    uint32_t da_gas{};
```

Comment:
This one should not be needed, or at least it might be better to do `= 0;`

Reply:
Good idea, I will assign 0 for each int primitive types struct members.